### ISSUE TITLE
Read page content after page change

### ISSUE THREAD

**Issue by PiotrKozlowski** [Connexions/webview#2013]
After changing page inside book there should be some kind of information for screen readers indicating content is loading and when it's done we should set `focus()` on page title.

After changing page with `enter` inside TOC or with `next` / `prev` buttons, focus should be at progress bar with message "Loading" and when content is loaded it should read the title.

<img width="1277" alt="screen shot 2018-10-26 at 12 12 02" src="https://user-images.githubusercontent.com/31478212/47572266-ddf5bc80-d93a-11e8-8184-8ac72bcc1c96.png">

![screenshot-katalyst01 cnx org-2018 10 26-12-12-50](https://user-images.githubusercontent.com/31478212/47572471-578daa80-d93b-11e8-8349-3ef8c885441e.png)

**Comment by uskka** [comment]
Works fine. Tested with Chromevox. 
https://drive.google.com/file/d/13iYS82RP8i98m4csTXUYV8jMrADXTQpw/view?usp=sharing
